assets/queries/cloudFormation/filter_and_alarm_exist_for_security_group_changes/query.rego

```diff
@@ -0,0 +1,74 @@
+package Cx
+
+CxPolicy [ result ] {
+
+	resourceMetricFilter := input.document[i].Resources
+    some name
+    resourceMetricFilter[name].Type == "AWS::Logs::MetricFilter"
+    
+    resourceAlarm := input.document[i].Resources
+    some nameAlarm
+    resourceAlarm[nameAlarm].Type == "AWS::CloudWatch::Alarm"
+   
+    checkAlarm(resourceMetricFilter[name].Properties.MetricTransformations[index].MetricNamespace, resourceAlarm[nameAlarm].Properties)
+
+    result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	     sprintf("Resources.%s.Properties.MetricTransformations.MetricNamespace", [name]),
```

Comment:
Since you don't/can't specify the array index, the search key will go to the first occurrence of 'MetricNamespace' in the array, which might not be the one you were aiming to. If you can't specify it, just point the key to the array itself